Blahgger/Blahgger/ApplicationInsights.config

```diff
@@ -0,0 +1,84 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<ApplicationInsights xmlns="http://schemas.microsoft.com/ApplicationInsights/2013/Settings">
```

Comment:
When creating projects, be sure you've unchecked the "Application Insights" option. If you're unsure of how to do this, let me know and I'll show you the next time you create a project.